src/main/java/com/spotinst/sdkjava/model/SpotOceanK8sClusterRepo.java

```diff
@@ -77,4 +82,47 @@
 
         return retVal;
     }
+
+    @Override
+    public RepoGenericResponse<List<OceanK8sCluster>> getAllK8sClusters(String authToken, String account) {
+        RepoGenericResponse<List<OceanK8sCluster>> retVal;
+
+        try {
+
+            List<ApiOceanK8sCluster> getClusters = SpotOceanK8sClusterService
```

Comment:
rename getClusters variable to getAllClustersApi

Reply:
updated